extensions/cornerstone-dicom-seg/src/viewports/OHIFCornerstoneSEGViewport.tsx

```diff
@@ -344,7 +344,9 @@ function OHIFCornerstoneSEGViewport(props) {
             patientSex: PatientSex || '',
             patientAge: PatientAge || '',
             MRN: PatientID || '',
-            thickness: SliceThickness ? `${parseFloat(SliceThickness).toFixed(2)}mm` : '',
+            thickness: SliceThickness
```

Comment:
about the mm, this is for the studyData in below

![image](https://github.com/OHIF/Viewers/assets/7490180/773d40f5-1343-4b31-bd6b-16e8da137adc)

what do you mean return as units

Reply:
> about the mm, this is for the studyData in below
> 
> ![image](https://user-images.githubusercontent.com/7490180/259114264-773d40f5-1343-4b31-bd6b-16e8da137adc.png)
> 
> what do you mean return as units

I mean have the units as a separate return value so that the formatting is up to the GUI, and isn't done in the data provided to the child element.
thickness: SliceThickness,
thicknessUnits: 'mm',

Then, when the thickness gets used for display, it can call roundNumber(thickness) + thicknessUnits to display both values.